docs/src/examples/components/Toolbar/Types/ToolbarExampleEditor.shorthand.tsx

```diff
@@ -129,7 +131,36 @@ const ToolbarExampleShorthand = () => {
         { key: 'bullets', icon: { name: 'bullets', outline: true } },
         { key: 'number-list', icon: { name: 'number-list', outline: true } },
         { key: 'divider3', kind: 'divider' },
-        { key: 'more', icon: { name: 'more', outline: true } },
+        {
+          key: 'more',
+          icon: { name: 'more', outline: true },
+          active: moreMenuOpen,
+          menu: {
+            items: [
+              {
+                content: 'Quote',
+                icon: 'quote',
+                onClick: () => alert('... -> Quote'),
+              },
+              {
+                content: 'Link',
+                icon: 'link',
+                disabled: true,
+                onClick: () => alert('SHOULD NOT BE CALLED, ITEM IS DISABLED... -> Link'),
+              },
+              {
+                content: 'Code snippet',
+                icon: 'code-snippet',
+                onClick: () => alert('... -> Code snippet'),
+              },
+            ],
+          },
+          menuOpen: moreMenuOpen,
+          onMenuOpenChange: (e, { menuOpen }) => {
+            console.log(`setting menu to ${menuOpen ? 'open' : 'close'}`)
```

Comment:
If we will use knobs I believe that we can't remove this.

And I think that we should never use `console.log()` in our examples because it's an item that is hidden from user until dev console is opened.

Reply:
Changed to knobs, but, still, this needs to be here to close the menu when it wants to be closed.